charts/kubeclarity/templates/deployment.yaml

```diff
@@ -9,12 +9,14 @@
 {{- $dbPort := index .Values "kubeclarity-postgresql" "service" "ports" "postgresql" -}}
 {{- $dbUser := index .Values "kubeclarity-postgresql" "auth" "username" -}}
 {{- $dbName := index .Values "kubeclarity-postgresql" "auth" "database" -}}
+{{- $dbSslMode := index .Values "kubeclarity-postgresql" "auth" "sslMode" -}}
```

Comment:
can we change `dbSslMode` to `dbSSLMode`?

Reply:
Done!